.babelrc

```diff
@@ -13,7 +13,7 @@
       {
         "modules": false,
         "targets": {
-          "browsers": ["last 3 versions"]
+          "browsers": ["last 3 versions", "ie >= 11"]
```

Comment:
IIRC, isn't `ie >=11` included in `last 3 version`?

Reply:
Yeah, you're right actually. Silly me assumed that edge is the next version of IE. Going to fix it now